config/app.php

```diff
@@ -78,7 +78,7 @@
 	|
 	*/
 
-	'timezone' => env('TIMEZONE', 'UTC'),
+	'timezone' => env('TIMEZONE', date('T')),
```

Comment:
```suggestion
	'timezone' => env('TIMEZONE', date('e')),
```

We need the timezone identifier or a so-called level-3 timezone, e.g. `Europe/Paris`. A timezone abbreviation like "CEST" (level-2) or a simple time offset like "+02:00" does not suffice.

Reply:
But wasn't the default before just `UTC` which is the level-2 variant?

Reply:
Yes and it produced a warning in the diagnostics to change that.